src/js/utils.js

```diff
@@ -2,6 +2,17 @@
 
 /* global convertDate, stripHtmlTags, htmlDecode, chunkArray, escapeSpecialChars */
 
+/*
+ * Remove soft hyphens and replace &#8217; with apostrophe.
+ * @param {String} webpageString - A string with one or more soft hyphens or &#8217;.
```

Comment:
Probably better to rename `webpageString` to `inputStr`. In this case, try to pick a name that suits the function itself rather than the context you wanted to use this function.